python/gs/jit/optimize.py

```diff
@@ -1,2 +1,119 @@
-def optimize(ir):
-    pass
\ No newline at end of file
+import operator
```

Comment:
Looks like the optimization pass is also not very relevant to this FastGCN PR. Could you move it to another PR?

Reply:
done. Thery'are on https://github.com/czkkkkkk/graph_sampling/pull/15